apps/web/pages/settings/my-account/booking-limits.tsx

```diff
@@ -0,0 +1,134 @@
+import { useForm, Controller } from "react-hook-form";
+
+import { IntervalLimitsManager } from "@calcom/features/eventtypes/components/tabs/limits/EventLimitsTab";
+import SectionBottomActions from "@calcom/features/settings/SectionBottomActions";
+import { getLayout } from "@calcom/features/settings/layouts/SettingsLayout";
+import { validateIntervalLimitOrder, parseBookingLimit, classNames } from "@calcom/lib";
+import { APP_NAME } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import type { RouterOutputs } from "@calcom/trpc/react";
+import type { IntervalLimit } from "@calcom/types/Calendar";
+import { Button, Form, Meta, SkeletonContainer, SkeletonText, showToast, SettingsToggle } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const SkeletonLoader = ({ title, description }: { title: string; description: string }) => {
+  return (
+    <SkeletonContainer>
+      <Meta title={title} description={description} borderInShellHeader={false} />
+      <div className="border-subtle mt-6 flex flex-col rounded-lg border px-4 py-6 sm:px-6">
+        <SkeletonText className="h-6 w-1/4" />
+        <SkeletonText className="mt-2 h-6 w-2/5" />
+      </div>
+    </SkeletonContainer>
+  );
+};
+
+const BookingsView = ({ data }: { data: RouterOutputs["viewer"]["me"] }) => {
+  const { t } = useLocale();
+  const bookingsLimitFormMethods = useForm({
+    defaultValues: {
+      bookingLimits: data?.bookingLimits as IntervalLimit,
```

Comment:
```suggestion
      bookingLimits,
```

Reply:
Because of [this](https://github.com/calcom/cal.com/pull/14243#discussion_r1800225972) we don't need to do this any more.